apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+  alt: "A modal containing a heading, multiple lines of text ant two buttons placed below the modal heading.",
+  width: "600px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them to display important information that needs user’s action or dismissal.",
+            "Use them when the user needs to perform one specific task.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for multi-step flows."
+        ]
+    }
+} %}
+
+---
+
+## Full page modal
+
+Also known as Modal View (iOS) or Scaffold View (Android) are a surface that overlays the page’s main content entirely and is used to display information, gather input or confirm actions.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/modal/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/frame.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/frame-narrow.svg",
+  alt: "A surface that overlays the page’s main content entirely and displays a list of payment methods.",
+  width: "360px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them for container multi-step interactions.",
+            "Use them when the user needs to perform various tasks.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them purely to display information. <br/> Use a bottom sheet instead.",
+            "Don’t use them for a contextual interaction. <br/> Use a bottom sheet instead."
+        ]
+    }
+} %}
+
+---
+
+## Notification
+
+A notification is an element used to display brief messages, alerts, or updates to users within a user interface.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/notification/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/notification.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/notification-narrow.svg",
+  alt: "The warning variant of the notification component that contains one line of text.",
+  width: "872px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them to display information which sits close to other UI components.",
+            "Use them when the message needs to be persistent.",
+            "They are not triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for short-mid form content.",
```

Comment:
I'm not entirely sure what this means. Is it saying not to use in forms? You could also interpret this as saying to put long text inside it instead of short or medium length text.

Reply:
This block was removed.